Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -1373,6 +1373,18 @@ TextureCacheBase::AllocateTexture(const TCacheEntryConfig& config)
   return entry;
 }
 
+TextureCacheBase::TexPool::iterator
+TextureCacheBase::FindMatchingTextureFromPool(const TCacheEntryConfig& config)
+{
+  // Find a texture from the pool that does not have a frameCount of FRAMECOUNT_INVALID.
+  // This prevents a texture from being used twice in a single frame with different data,
+  // which potentially means that a driver has to maintain two copies of the texture anyway.
+  auto range = texture_pool.equal_range(config);
```

Comment:
@lioncash I don't see the point in having this as a separated function. Looking up in a pool is what I'm expecting on allocating something.